src/site/stages/prearchive/link-assets-to-bucket.js

```diff
@@ -6,15 +6,57 @@
  */
 
 /* eslint-disable no-continue */
+/* eslint-disable no-console */
 
 const fs = require('fs');
-const jsdom = require('jsdom');
 const path = require('path');
+const cheerio = require('cheerio');
 const buckets = require('../../constants/buckets');
 
 const TEAMSITE_ASSETS = 'va_files';
 
+function getAssetLinkElements(doc) {
+  const elementTags = 'script, img, link, picture > source';
+  return doc(elementTags);
+}
+
+function invalidAssetSrcCheck(assetSrc) {
+  return (
+    !assetSrc ||
+    assetSrc.startsWith('http') ||
+    assetSrc.startsWith('data:') ||
+    assetSrc.includes(TEAMSITE_ASSETS)
+  );
+}
+
+function updateAssetBucketLocation(prop, assetSrc, item, bucketPath) {
+  let assetBucketLocation;
+  if (prop === 'srcset') {
+    const sources = assetSrc.split(',');
+    assetBucketLocation = sources
+      .map(src => `${bucketPath}${src.trim()}`)
+      .join(', ');
+  } else {
+    assetBucketLocation = `${bucketPath}${assetSrc}`;
+  }
+  item.attr(prop, assetBucketLocation);
```

Comment:
Just realizing and now noting that this changed from `setAttribute` to `attr` because of Cheerio (I'm assuming).

Reply:
yes, this is because of Cheerio. Thanks for looking at it!